src/PowerShellEditorServices/Services/CodeLens/ICodeLensProvider.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.PowerShell.EditorServices.CodeLenses
     /// <summary>
     /// Specifies the contract for a Code Lens provider.
     /// </summary>
-    public interface ICodeLensProvider
+    internal interface ICodeLensProvider
```

Comment:
@TylerLeonhardt don't you use this one?  @rjmholt is there another API for registering custom code lenses?

Reply:
It doesn't look like we pick any of these up from anywhere at the moment, so even if you instantiate one of these, I'm don't think there's actually a way to register it.

Reply:
The types on the interface seem to have changed too, and now point to Omnisharp types. So we'll need to wrap them

Reply:
Fair enough.  Probably worth rethinking the approach anyway, they were particularly difficult to work with.